kolbot/PlugY.md

```diff
@@ -0,0 +1,37 @@
+[General Table of Contents](https://github.com/blizzhackers/documentation/#diablo-2-botting-system)
+
+[Kolbot Table of Contents](https://github.com/blizzhackers/documentation/tree/master/kolbot/#kolbot)
+
+---
+
+# PlugY Setup for Single Player botting
+
+---
+
+* [Installing PlugY](#installing-plugy)
+
+* [Configuring D2BS and kolbot for PlugY](#configuring-d2bs-and-kolbot-for-plugy-single-player)
+
+---
+
+## Installing PlugY
+
+1. Download PlugY from the [official website](http://plugy.free.fr/PlugY_The_Survival_Kit_v14.03.zip)
+
+2. Download the [PlugY D2FilePatcher](http://plugy.free.fr/PlugY_The_Survival_Kit_v14.03_D2FilePatcher.zip)
+
+3. Extract both ZIP files to your Diablo II folder.
+
+4. Run the PatchD2File.exe and select Yes.
+
+---
+
+## Configuring D2BS and Kolbot for PlugY single player.
+
+1. Go to \d2bs\kolbot\libs\core and open Config.js
+
+2. Find and set the line PlugyStash: true,

```

Comment:
It'd be better to have this set inside the character config file rather than globally. Users might run some profiles on plugy and some not